### PR TITLE
Proposed improvement for shift mailing

### PR DESCRIPTION
**Pull request by pitpalme**
Excel sheet is now more detailed, but also hopefully more clear to read.

Staff and security at the only location currently making use of this feature would benefit from new "layout". The idea is to shorten lines of Volunteers being registered when entering facility by having names already printed in significant order.

### CODE REVIEW

File: shiftmailer/excelexport.py [hunk 1]
```diff
@@ -1,61 +1,148 @@
...
-import time
+import logging
+import os
+import tempfile
 
 from django.core.mail.message import EmailMessage
 import xlwt
-from django.db.models import Count
+
+log = logging.getLogger(__name__)
```
Comment (maikhoepfel) [inline shiftmailer/excelexport.py]: This is a nitpick, but `logger = logging.getLogger(__name__)` is, if I'm not mistaken, the much more common pattern.

**Comment by maikhoepfel** [comment]
I don't have good sample data and hence haven't run the command. But the code changes look good to me, so if the Excel output is indeed better, please feel free to merge at your convenience.

**Comment by christophmeissner** [comment]
Thanks, looks good for me.

**Comment by pitpalme** [comment]
Thanks Maik,

I'll keep that in mind for further development and adjust it for shiftmail next time I touch the code :)
Nitpicking is OK, as I'm new to python and django and therefore need it to learn how to code it correctly. :D

**Pull request merged**